easybuild/easyconfigs/m/magma/magma-2.0.0.patch

```diff
@@ -0,0 +1,106 @@
+diff -ru magma-2.0.0.orig//Makefile magma-2.0.0/Makefile
```

Comment:
please add a comment on top of this patch explaining what it does, and who the author is